models/personel.py

```diff
@@ -0,0 +1,21 @@
+class Personel(object):
+    """personel model"""
+
+    def __init__(self, name):
+        self.name = name
+        want_accommodation = 'N'
+
+
+class staff(Personel):
+
+    def __init__(self, name):
```

Comment:
**Code Quality: Nitpick**

You should take advantage of the superclass constructor to ensure variables like `wants_accomodation` are actually set on object creation.